druid-shell/src/platform/gtk/clipboard.rs

```diff
@@ -64,9 +64,17 @@ impl Clipboard {
             let idx = idx as usize;
             if idx < formats.len() {
                 let item = &formats[idx];
+                let mut not_set = true;
                 if item.identifier == ClipboardFormat::TEXT {
```

Comment:
I think you can do
```rust
if let (ClipboardFormat::TEXT, Ok(data)) = (item.identifier, std::str::from_utf(&item.data)) {
    sel.set_text(data);
} else {
    //...
}
```
and then you don't need the `not_set` logic.

Reply:
Good idea, done.

Note however that this might always call `from_utf8`, even with stuff other than `ClipboardFormat::TEXT`.